spec/spec_helper.rb

```diff
@@ -6,7 +6,11 @@ def input(model, field)
   end
 
   def visit(page_name)
-    page.visit("/#{page_name}.html")
+    if I18n.locale == :es
+      page.visit("/#{page_name}_#{I18n.locale}.html")
```

Comment:
ah. This may be a little more clever than we need. Can we rely on explicitly visiting the `_es` variant instead, please?